### ISSUE TITLE
Machine controller requires looking at it with screwdriver to activate pump active status on a pipe

### ISSUE THREAD

**Issue by eragaxshim** [GregTechCEu/GregTechCEu-1.19#2065]
### Checked for existing issues

- [X] I have checked for existing issues, and have found none.

### Tested latest version

- [X] I have checked that this occurs on the latest version.

### GregTech CEu Version

1.4.4

### Minecraft Version

1.20.1

### Recipe Viewer Installed

JEI

### Environment

Single Player

### Cross-Mod Interaction

No

### Other Installed Mods

Gregtech Community Pack

### Expected Behavior

When activating the machine controller with redstone, you expect it to immediately change the state of the pump that is installed on the same pipe. 

### Actual Behavior

The pump only starts (or stops) pumping when you look at it with the screwdriver, after you change the redstone state.

### Steps to Reproduce

1. Place two tanks, connect them with a pipe. Place a pump on the pipe, set it to import from the tank and put a machine controller on the same block. 
2. Fill up one of the tanks. The pump will start pumping fluid.
3. Activate the machine controller with redstone, like a lever.

The pump will continue to pump. But when you open the controller with a screwdriver, it will stop. If you flick the switch, nothing will happen until you open the controller interface, after which it will start pumping again.

### Additional Information

I don't have the time to investigate the cause but putting this here so I don't forget.

**Comment by meza** [comment]
Running into this on 1.6.4

**Comment by jurrejelle** [comment]
1.6.4 is almost a year old now, 7.5.2 is our latest release. We don't provide support for versions that old

**Comment by meza** [comment]
Yeah I figured. Stuck on it until Star Technology updates sadly

**Comment by jurrejelle** [comment]
> Yeah I figured. Stuck on it until Star Technology updates sadly

Yeah, fair enough. Unfortunately even if we fix bugs, we can't force packs to update.

**Comment by meza** [comment]
All good. Just figured if this is still open, it might still be an issue on modern versions. Here's to hoping it's not :) It's making stocking fluids quite difficult :D

**Comment by Wmeje0** [comment]
This is still an issue on 7.5.2, also with conveyors and other covers
